Lesson_5/src/com/cycles/exercise12.java

```diff
@@ -0,0 +1,32 @@
+package com.cycles;
+
+public class exercise12 {
+    public static void main(String[] args) {
+
+        // найти сумму первых Н целых чисел которые делятся на 3
+
+
+        int number = 12;
+        int b = 1, c = 0;
+        int count = 1;
+        System.out.println("число " + number);
```

Comment:
1. Too complicated
2. Not correct. If I need to get sum of first 3 numbers that could be devided on 3 I expect to get 18 (3 + 6 + 9). But in your case I'll get 3 as a result. Correct please